packages/terra-layout/tests/jest/__snapshots__/LayoutSlidePanel.test.jsx.snap

```diff
@@ -21,6 +21,7 @@ exports[`LayoutSlidePanel should render when small 1`] = `
       isRelativeToContainer={true}
```

Comment:
Coincidentally this package already has a change log update titled "Regenerate jest snapshots" from a similar issue resolved on this pull request: https://github.com/cerner/terra-framework/pull/325

Because the change log update would be identical I didn't add one for this fix.